src/OmniSharp.Host/CommandLineApplication.cs

```diff
@@ -80,7 +80,7 @@ private void DebugAttach()
         {
             if (Debug)
             {
-                Console.WriteLine($"Attach debugger to process {Process.GetCurrentProcess().Id} to continue...");
+                //Console.WriteLine($"Attach debugger to process {Process.GetCurrentProcess().Id} to continue...");
```

Comment:
Oops.. This should not be here.

BTW, @david-driscoll, we need a way to get this working with LSP? Currently VSCode starts yelling at me when non-json-RPC formatted content arrives on STDOUT.

Reply:
I'd be fine with it commented for now... worst case we can always have someone uncomment it for debugging purposes, if they haven't figure out which pid they need.